plugin-api/src/main/java/org/hyperledger/besu/plugin/services/query/PoaQueryService.java

```diff
@@ -36,4 +36,11 @@
    * @return The identity of the proposer for the given block.
    */
   Address getProposerOfBlock(final BlockHeader header);
+
+  /**
+   * Retrieves the signer {@link Address} of the local node.
```

Comment:
You are not really adding any detail beyond that already provided by the method signature.
Perhaps these questions might be worth covering in the Javadoc:
- What is this local signer address in the context of the node?
- What is this local signer address in the context of the network?
- What is the intended use / would it be used?

Reply:
👍